cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetadataStoreDataset.java

```diff
@@ -365,6 +360,14 @@ public void deleteAll(MDSKey id, @Nullable Predicate<MDSKey> filter) {
     }
   }
 
+  public void delete(MDSKey id) {
+    try {
+      table.delete(id.getKey(), COLUMN);
```

Comment:
No specific reason, looks like all operations were being performed on that column. So just maintained that for uniformity.